### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stratusceo**
Providing more information about how Base Nodes work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,33 @@ Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the ne
...
 This repository contains the relevant Docker builds to run your own node on the Base network.
 
-### Requirements
+### Hardware requirements
 
 We recommend you this configuration to run a node:
 - at least 16 GB RAM
 - an SSD drive with at least 100 GB free
 
+### Troubleshootings
```
Comment (wbnns) [inline README.md:15]: s/Troubleshootings/Troubleshooting/

File: README.md [hunk 2]
```diff
@@ -6,19 +6,33 @@ Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the ne
...
+
+If you encounter problems with your node, please reach it out on [our Discord](https://discord.gg/buildonbase):
```
Comment (wbnns) [inline README.md:17]: s/please reach it out/please open a GitHub issue or reach out/

Note: You could also link `GitHub issue` to https://github.com/base-org/node/issues/new/choose

File: README.md [hunk 3]
```diff
@@ -6,19 +6,33 @@ Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the ne
...
+- take the "Developer" role (to do this, go to `the server menu > Linked Roles > connect GitHub`)
```
Comment (wbnns) [inline README.md:18]: Maybe rephrase to:

- Once you've joined, in the Discord app go to `server menu` > `Linked Roles` > `connect GitHub` and connect your GitHub account so you can gain access to our special developer channels

File: README.md [hunk 4]
```diff
@@ -6,19 +6,33 @@ Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the ne
...
+- ask it in `#🛟|node-support`
```
Comment (wbnns) [inline README.md:19]: Maybe rephrase to:

- Report your issue in `#🛟|node-support`

File: README.md [hunk 5]
```diff
@@ -6,19 +6,33 @@ Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the ne
...
+
 ### Supported networks
 
 | Ethereum Network | Status |
 |------------------|--------|
 | Goerli testnet   | ✅     |
 | Mainnet          | 🚧     |
 
+### Upgrades
```
Comment (wbnns) [inline README.md:28]: s/Upgrades/Upgrading your node

Additional nit -- if you haven't already done so, we should confirm this as the recommended procedure for upgrading a Base node.

File: README.md [hunk 6]
```diff
@@ -6,19 +6,33 @@ Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the ne
...
+
+If you run a node you need to subscribe to [an update feed](https://community.optimism.io/docs/developers/releases/) (either [the mailing list](https://groups.google.com/a/optimism.io/g/optimism-announce) or [the RSS feed](https://changelog.optimism.io/feed.xml)) to know when to upgrade. <br/>
+Otherwise, your node will eventually stop working.
+
+### Hosted Goerli L1 nodes providers
```
Comment (wbnns) [inline README.md:33]: Since this requirement is already mentioned in the first step of the `Usage` section that immediately follows this new text, was thinking maybe should consider removing this and just linking `Goerli L1 node RPC` in that first sentence to https://docs.base.org/tools/node-providers instead.

**Review by wbnns — COMMENTED** [review]
@stratusceo 

Hello. 👋 I went through and saw a few things that could potentially be revised, if welcome.

**Comment by stratusceo** [comment]
I've made a new PR with the edits, that really looks better like this!

**Comment by wbnns** [comment]
@stratusceo

Nice! Glad to see that the updated one you made, just got merged! 👍️
